synapse/lib/types.py

```diff
@@ -481,6 +484,45 @@ def indxByEq(self, valu):
 
         return Type.indxByEq(self, valu)
 
+
+class Range(Type):
+
+    _opt_defs = {
+        ('type', None),
+    }
+
+    def postTypeInit(self):
+        tdef = self.opts.get('type')
+        if not(type(tdef) is tuple and len(tdef) is 2):
+            raise s_exc.BadTypeDef(self.opts)
+
+        try:
+            self.tdef = self.modl.type(tdef[0]).clone(tdef[1])
```

Comment:
small nit, but the (name, info) tuple is being called a "type def", and most instances of type objects are being called _type ( or in this case "subtype" would probably be fine).  Just trying to avoid confusion over what a "tdef" is :D

Reply:
makes sense

Reply:
fixed